sources/packages/forms/src/form-definitions/sfaa2023-24.json

```diff
@@ -30565,7 +30746,7 @@
                   "tags": [],
                   "conditional": {
                     "show": "true",
-                    "when": "eligibleForAnAdditionalTransportationAllowance",
+                    "when": "additionalTransportListedDriver",
```

Comment:
I know the existing code was applying the hide/show condition to every question, but the condition is at the panel level and we do not have to repeat the same condition to every item inside the panel unless I am missing something. 

![image](https://github.com/user-attachments/assets/2e164234-a17a-458b-afea-d37f336b71d5)